File: WeatherServer/WeathARServer/src/main/java/de/htw/mm/weathar/model/WeatherData.java
package de.htw.mm.weathar.model;

public class WeatherData {

    private String temperature;

    private DayTime dayTime;

    private WeatherPhenomena phenomena;

    public String getTemperature() {
        return temperature;
    }

    public void setTemperature(String temperature) {
        this.temperature = temperature;
    }

    public DayTime getDayTime() {
        return dayTime;
    }

    public void setDayTime(DayTime dayTime) {
        this.dayTime = dayTime;
    }

    public WeatherPhenomena getPhenomena() {
        return phenomena;
    }

    public void setPhenomena(WeatherPhenomena phenomena) {
        this.phenomena = phenomena;
    }
}

File: WeathAR/Assets/Scripts/Model/WeatherData.cs
using System;

namespace Model
{
    /**
     * Representation of the weather data construct. Is used as a wrapper object for the response json.
     *
     * @author Cong Minh Nguyen, Tuan Tung Tran
     * @date 20.07.2019
     * 
     */
    [Serializable]
    public class WeatherData
    {
        public double temperature;

        public DayTime dayTime;

        public WeatherPhenomena phenomena;
    }
}